xdebug_stack.c

```diff
@@ -1383,6 +1402,19 @@ function_stack_entry *xdebug_add_stack_frame(zend_execute_data *zdata, zend_op_a
 	return tmp;
 }
 
+void xdebug_remove_stack_frame(function_stack_entry *fse) {
+	size_t i = 0;
+	/* Unlocks the garbage collection of stack frame (function call arguments) objects
+	 * So these objects can be properly GCed by PHP later on */
+	for (i = 0; i < fse->gc_locked_objects_count; i++) {
+		#if PHP_VERSION_ID >= 70300
```

Comment:
Please move the `#if …` to column 1 again, and don't extra indent `GC_DELREF/GC_REFCOUNT`.

Reply:
Done